bin/s3downloader.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022
+#
+# Author(s):
```

Comment:
Skip this part and add "Pytroll Developers" to the copyright. An AUTHORS.MD file should eventually be added. You could add it if you like now and close the issue I recently opened.